bundle/manifests/metallb.io_addresspools.yaml

```diff
@@ -42,7 +42,7 @@ spec:
                 items:
                   type: string
                 type: array
-              autoAssign:
+              auto-assign:
```

Comment:
I'm surprised this changed in this direction? This doesn't seem to go along with typical kube API conventions?

Reply:
That changed in https://github.com/metallb/metallb-operator/pull/39 to make the conversion to configmap easier IIRC, we can revert that and add a conversion layer.